code/modules/clothing/modular_armor/attachments/secondary_color.dm

```diff
@@ -34,20 +44,20 @@
 	current_variant = "normal"
 	icon_state_variants = list(
 		"normal",
-		"webbing",
+		"hardsuit webbing",
```

Comment:
just undo this change and linters will be fixed

Reply:
fixed